transmart-notifications/grails-app/controllers/org/transmartproject/notifications/SubscriptionMailController.groovy

```diff
@@ -0,0 +1,36 @@
+package org.transmartproject.notifications
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.RequestParam
+import org.transmartproject.core.exceptions.InvalidArgumentsException
+import org.transmartproject.core.userquery.SubscriptionFrequency
+
+class SubscriptionMailController {
```

Comment:
Code looks really nice. Simple and well documented. But perhaps this controller can be named NotificationMailController or NotificationController, because it does not mail subscriptions.

Reply:
true, I wanted it to be in line with QuerySetSubscriptionMailService, I guess more renaming is needed then